server/db/models/tests.js

```diff
@@ -15,7 +15,7 @@ const Test = db.define('test', {
   }
 });
 
-Test.passing = function() {
+Test.passing = function () {
```

Comment:
Ideally since this is a method, it should be named as such, and maybe be something like `getPassing`? Right now `passing` is not clear when used... reads like it might return a boolean

Reply:
Agreed. Updated!